creator-node/src/snapbackSM/syncHistoryAggregator.js

```diff
@@ -160,6 +171,36 @@ class SyncHistoryAggregator {
     return currentLatestSyncData
   }
 
+  /**
+   * Returns the number of unique users with successful and fail syncs that day. Will be `null` if a sync with those
+   * states have not been triggered.
+   * @param {Object?} logContext the log context off of the Express req object
```

Comment:
can we update js doc comment with date param?

Reply:
yep!